test/snowpack/config/buildOptions.baseUrl/index.test.js

```diff
@@ -0,0 +1,60 @@
+const {testFixture} = require('../../../fixture-utils');
+
+describe('buildOptions.baseUrl', () => {
+  beforeAll(() => {
+    // Needed until we make Snowpack's JS Build Interface quiet by default
+    require('snowpack').logger.level = 'warn';
+  });
+
+  describe.each([
```

Comment:
one of the reasons I'd like to lean into limited snapshot testing here: we can support testing multiple config values with the same, full test suite without having to change some inline matching string.